rpc/src/v1/traits/eth.rs

```diff
@@ -31,7 +31,7 @@ build_rpc_trait! {
 		#[rpc(name = "eth_protocolVersion")]
 		fn protocol_version(&self) -> Result<String>;
 
-		/// Returns an object with data about the sync status or false. (wtf?)
+		/// Returns an object with data about the sync status or false.
```

Comment:
out-of-scope IMO but I couldn't resist. 

we should not such comments in the code base :small_airplane: